### ISSUE TITLE
objectness / confidence scores

### ISSUE THREAD

**Issue by mkilicar** [AntonMu/TrainYourOwnYOLO#217]
Is the intersection over union (iou(prediction, target)) considered when computing the confidence scores at the training time?
It seems to me iou is used just for ignoring some boxes but it doesn't look how good/bad is the bbox is.
Am I correct on this? Can you a little bit elaborate object_mask? Thanks

https://github.com/AntonMu/TrainYourOwnYOLO/blob/50781251c1d82669ee3306f8b29ce44d0e514070/2_Training/src/keras_yolo3/yolo3/model.py#L492

"The confidence score reflects how likely the box contains an object (objectness) and how accurate is the boundary box."

**Comment by AntonMu** [comment]
The loss function for training is the Yolov3 loss function. You can find the details in the Yolov3 paper. There are also some explanations online.
